tests/labhub_test.py

```diff
@@ -326,7 +325,7 @@ def test_invite_me(self):
 
         plugins.labhub.os.environ['GH_TOKEN'] = 'patched?'
         testbot.assertCommand('!invite me',
```

Comment:
This should be `hello world`

Reply:
ughh. we don't need this test case at all